draft-ietf-mls-protocol.md

```diff
@@ -2078,10 +2073,10 @@ struct {
 } MLSSenderDataAAD;
 ~~~~~
 
-When parsing a SenderData struct as part of message decryption, the
-recipient MUST verify that the sender field represents an occupied
-leaf in the ratchet tree.  In particular, the sender index value
-MUST be less than the number of leaves in the tree.
+When parsing a SenderData struct as part of message decryption, if the
+SenderType is `member`,  the recipient MUST verify that the sender field
+represents an occupied leaf in the ratchet tree.  In particular, the sender
+index value MUST be less than the number of leaves in the tree.
```

Comment:
Maybe mark the `sender` field in `MLSSenderData` as of type `// leaf index`?

Reply:
Good idea.  I think this paragraph is unnecessary anyway.